proto/storage/kube_event.proto

```diff
@@ -24,11 +24,16 @@ message KubernetesEvent {
     message Object {
         // Resource and sub-resource mapping is represented as resource_subresource.
         enum Resource {
-            UNKNOWN          = 0;
-            PODS_EXEC        = 1;
-            PODS_PORTFORWARD = 2;
-            SECRETS          = 3;
-            CONFIGMAPS       = 4;
+            UNKNOWN                    = 0;
+            PODS_EXEC                  = 1;
+            PODS_PORTFORWARD           = 2;
+            SECRETS                    = 3;
+            CONFIGMAPS                 = 4;
+            CLUSTERROLES               = 5;
+            CLUSTERROLEBINDINGS        = 6;
+            NETWORKPOLICIES            = 7;
+            SECURITYCONTEXTCONSTRAINTS = 8;
+            EGRESSFIREWALLS            = 9;
```

Comment:
Some related questions:
* Is there a reason why new types in alert.proto **do** have underscore but these **do not**?
* Are the changes to frontend alert.proto.ts which **do not** have underscore correct or incorrect?
* Which of these backend enum types corresponds to policyCriteriaDescriptors.tsx file?

In favor of underscore convention: Instead of need to touch so many frontend files when under pressure, UI might be able to **compute** text and abbreviation of unexpected types:
* text: replace underscore with space and upper with lower: cluster role bindings
* abbreviation: first letter and first letter after any underscore: CRB